frontend/src/scenes/groups/GroupsIntroduction.tsx

```diff
@@ -42,19 +42,26 @@ export function GroupsIntroduction({ access }: Props): JSX.Element {
     )
 }
 
-export function GroupIntroductionFooter(): JSX.Element {
+export function GroupIntroductionFooter({ access }: Props): JSX.Element {
+    const needsToUpgrade = [GroupsAccessStatus.HasGroupTypes, GroupsAccessStatus.NoAccess].includes(access)
     return (
         <div className="text-sm bg-side rounded p-2" style={{ maxWidth: '15rem' }}>
-            Enter your payment information to use group analytics.{' '}
-            <Link
-                className="font-medium"
-                to="/organization/billing"
-                target="_blank"
-                data-attr="group-analytics-upgrade"
-            >
-                Upgrade
-            </Link>{' '}
-            or{' '}
+            {needsToUpgrade ? (
+                <>
+                    Enter your payment information to use group analytics.{' '}
```

Comment:
I've also proposed changing the copy of this in https://github.com/PostHog/posthog/pull/12377/files#diff-8f7d1ad363cd01229e6562df6405d00cdbd7479c9fc4652a99a4b27ddf563d86, but if you feel "Enter your payment information to use group analytics." is more effective, I'll keep it there as well

Reply:
The problem is that there should be a distinction between `GroupsAccessStatus.HasGroupTypes, GroupsAccessStatus.NoAccess` and `GroupsAccessStatus.HasAccess`.

The first two refer to a user who needs to upgrade, the third one to a user who has upgraded but hasn't used group analytics yet.

Reply:
Yeah, I saw that too, so also in that PR I made it so that `GroupsIntroductionFooter` is only shown on `NoAccess` and `HasGroupTypes` (and not `HasAccess`), as I didn't know you're about to pick this up. 😄 That can be adjusted too.

Reply:
If you already have it underway, you can maybe just add a message for the `HasAccess` one, so that people are redirected to the guide, like I did, and we can close this PR.

Reply:
Will do that and tag you!